src/lib/comms/Makefile

```diff
@@ -121,12 +124,21 @@ obj-networking-$(MAVLINK)-extra-cflags += \
 obj-networking-$(LWM2M) += \
 	sol-lwm2m.o
 
+obj-bluetooth-$(BLUETOOTH) += \
+	sol-bluetooth.o
+
+obj-bluetooth-$(PLATFORM_LINUX) += \
```

Comment:
see what I did for crypto, that way you select the one for your platform and your makefile will be as simple as now, just a different symbol.